packages/block-editor/src/components/letter-spacing-control/index.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { TextControl } from '@wordpress/components';
+import { ZERO } from '@wordpress/keycodes';
+
+/**
+ * Internal dependencies
+ */
+import {
+	BASE_DEFAULT_VALUE,
+	RESET_VALUE,
+	STEP,
+	isLetterSpacingDefined,
+} from './utils';
+
+const MIN_LETTER_SPACING = -5;
+const MAX_LETTER_SPACING = 10;
+
+export default function LetterSpacingControl( {
```

Comment:
This is taken 1:1 from the [line-height control](https://github.com/WordPress/gutenberg/tree/4b21017376808bf33a1cf23b3f85ec09e7f6443d/packages/block-editor/src/components/line-height-control) component. 

I think there could be scope to abstract/combine them, especially if we plan to freshen up the design of the sidebar controls (See https://github.com/WordPress/gutenberg/issues/27331)